example/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "rimraf dist/js && rollup --config"
   },
   "devDependencies": {
-    "@wasm-tool/rollup-plugin-rust": "^1.0.0",
+    "@wasm-tool/rollup-plugin-rust": "file:..",
```

Comment:
I prefer for the example to be self-contained, so it will work even if the user copy-pastes it into their own project.

Reply:
(Also note that if you want to test locally, you can use [`yarn link`](https://classic.yarnpkg.com/en/docs/cli/link) to override `@wasm-tool/rollup-plugin-rust` with the local version)